app/code/Magento/Checkout/view/frontend/web/js/shopping-cart.js

```diff
@@ -14,14 +14,14 @@ define([
         _create: function () {
             var items, i, reload;
 
-            $(this.options.emptyCartButton).on('click', $.proxy(function (event) {
-                if (event.detail === 0) {
-                    return;
-                }
-
+            $(this.options.emptyCartButton).on('click', $.proxy(function () {
                 $(this.options.emptyCartButton).attr('name', 'update_cart_action_temp');
                 $(this.options.updateCartActionContainer)
                     .attr('name', 'update_cart_action').attr('value', 'empty_cart');
+
+                if ($(this.options.emptyCartButton).parents('form').length > 0) {
```

Comment:
Can you describe why this fixes the reported bug?

Reply:
@DrewML: I remove `event.detail == 0 ` because it isn't supported by IE.11.

`event.detail` code is added to fix the bug https://github.com/magento/magento2/issues/21499. 

I removed it and changed to other solution to fix issue 21499. When the cart is updated, it should't call the emptyCartButton. So I change it to "button" element. So the emptyCartButton isn't trigger click when the qty is update.